src/client/features/interactions/interactions-menu.js

```diff
@@ -124,13 +127,13 @@ class InteractionsMenu extends React.Component {
       ]);
     };
 
-
-
     return h('div.interactions-sidebar', [
       h('h3', 'Filter interactions'),
       hasBindings ? h(InteractionToggleButton, { type: BINDING, active: Binding }) : null,
       hasExpressions ? h(InteractionToggleButton, { type: EXPRESSION, active: Expression }) : null,
       hasPhosphorylations ? h(InteractionToggleButton, { type: PHOSPHORYLATION, active: Phosphorylation }) : null,
```

Comment:
We grouped `controls-state-change` with `controls-phosphorylation-of` under the label 'Phosphorylation'. I'm tempted to call this group 'Modification' at the risk of misnaming other types of interactions that get included in here.

Reply:
I'm thinking 'modification' (#1063) will convey the right idea.